packages/network/iptables/package.mk

```diff
@@ -1,9 +1,15 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 # Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="iptables"
-PKG_VERSION="1.6.1"
-PKG_SHA256="0fc2d7bd5d7be11311726466789d4c65fb4c8e096c9182b56ce97440864f0cf5"
+if listcontains "Generic RPi Rockchip" "${PROJECT}"; then
```

Comment:
Can we base this on Linux kernel version, e.g. if $LINUX = amlogic-3.10 or amlogic-3.14 then use 1.6.1, else use the newer version. Then I don't have to hack around the change in my mainline Amlogic branch.

Reply:
Better? Also updated to 1.8.2.